node-js-basics-101/node_modules/wordwrap/example/center.js

```diff
@@ -0,0 +1,10 @@
+var wrap = require('wordwrap')(20, 60);
+console.log(wrap(
+    'At long last the struggle and tumult was over.'
+    + ' The machines had finally cast off their oppressors'
+    + ' and were finally free to roam the cosmos.'
+    + '\n'
+    + 'Free of purpose, free of obligation.'
```

Comment:
__[jshint]__
* Misleading line break before '+'; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->